back-end/api/quickstart/models.py

```diff
@@ -1,13 +1,43 @@
 from django.db import models
 from django.contrib.auth.models import User
+from django.utils import timezone
+
+SHORT_CHAR_LENGTH = 32
+LONG_CHAR_LENGTH = 128
 
 # Create your models here.
 class Author(models.Model):
   # TODO: uncomment this when we want to use built in User model behaviour (probably for authorization)
   # user = models.OneToOneField(User,on_delete=models.CASCADE) 
-  _id = models.CharField(max_length=128, default='', unique=True)
+  _id = models.CharField(max_length=LONG_CHAR_LENGTH, unique=True)
   _type = 'author'
-  displayName = models.CharField(max_length=32, default='')
-  url = models.CharField(max_length=128, default='', unique=True)
-  github = models.CharField(max_length=128, default='')
-  
\ No newline at end of file
+  displayName = models.CharField(max_length=SHORT_CHAR_LENGTH)
+  url = models.CharField(max_length=LONG_CHAR_LENGTH, unique=True)
+  github = models.CharField(max_length=LONG_CHAR_LENGTH)
+
+
+class Post(models.Model):
+
+  class Visibility(models.TextChoices):
+    PUBLIC = 'P'
+    FRIENDS = 'F'
+
+  _id = models.CharField(max_length=LONG_CHAR_LENGTH, unique=True)
+  _type = 'post'
+  title = models.CharField(max_length=LONG_CHAR_LENGTH, unique=True)
+  description = models.TextField()
+  source = models.CharField(max_length=LONG_CHAR_LENGTH, unique=True)
+  origin = models.CharField(max_length=LONG_CHAR_LENGTH, unique=True)
+  visibility = models.CharField(max_length=SHORT_CHAR_LENGTH, choices=Visibility.choices)
+  unlisted = models.BooleanField()
+  isPrivateToFriends = models.BooleanField()
+  author = models.JSONField()
```

Comment:
Should author just be an author ID (URL) instead? It seems like you're treating it as an actual author object, but we might have to get authors from other sites right?

Reply:
Fixed